modules/otel-forwarding-logs-to-tempostack.adoc

```diff
@@ -99,80 +94,51 @@ spec:
-          - sources:
-              - from: connection
+      k8sattributes: {}
       resource:
         attributes: # <1>
-          - key: loki.format # <2>
-            action: insert
-            value: json
-          - key:  kubernetes_namespace_name
+          - key:  kubernetes.namespace_name
             from_attribute: k8s.namespace.name
             action: upsert
-          - key:  kubernetes_pod_name
+          - key:  kubernetes.pod_name
             from_attribute: k8s.pod.name
             action: upsert
-          - key: kubernetes_container_name
+          - key: kubernetes.container_name
             from_attribute: k8s.container.name
             action: upsert
           - key: log_type
             value: application
             action: upsert
-          - key: loki.resource.labels # <3>
-            value: log_type, kubernetes_namespace_name, kubernetes_pod_name, kubernetes_container_name
-            action: insert
       transform:
         log_statements:
           - context: log
             statements:
               - set(attributes["level"], ConvertCase(severity_text, "lower"))
     exporters:
-      loki:
-        endpoint: https://logging-loki-gateway-http.openshift-logging.svc.cluster.local:8080/api/logs/v1/application/loki/api/v1/push # <4>
+      otlphttp:
+        endpoint: https://logging-loki-gateway-http.openshift-logging.svc.cluster.local:8080/api/logs/v1/application/otlp
+        encoding: json
         tls:
           ca_file: "/var/run/secrets/kubernetes.io/serviceaccount/service-ca.crt"
         auth:
           authenticator: bearertokenauth
       debug:
         verbosity: detailed
     service:
-      extensions: [bearertokenauth] # <5>
+      extensions: [bearertokenauth] # <2>
       pipelines:
         logs:
           receivers: [otlp]
           processors: [k8sattributes, transform, resource]
-          exporters: [loki] # <6>
+          exporters: [otlphttp] # <3>
         logs/test:
           receivers: [otlp]
           processors: []
           exporters: [debug]
 ----
-<1> Provides the following resource attributes to be used by the web console: `kubernetes_namespace_name`, `kubernetes_pod_name`, `kubernetes_container_name`, and `log_type`. If you specify them as values for this `loki.resource.labels` attribute, then the Loki Exporter processes them as labels.
-<2> Configures the format of Loki logs. Supported values are `json`, `logfmt` and `raw`.
-<3> Configures which resource attributes are processed as Loki labels.
-<4> Points the Loki Exporter to the gateway of the LokiStack `logging-loki` instance and uses the `application` tenant.
-<5> Enables the BearerTokenAuth Extension that is required by the Loki Exporter.
-<6> Enables the Loki Exporter to export logs from the Collector.
+<1> Provides the following resource attributes to be used by the web console: `kubernetes.namespace_name`, `kubernetes.pod_name`, `kubernetes.container_name`, and `log_type`.
```

Comment:
Are these literally e.g. `kubernetes.namespace_name`, or are these user-replaced values (e.g. `kubernetes.my-important-namespace`)? If the latter, consider using the user-replaced value convention: `kubernetes.<namespace_name>`

If these are literal please disregard my comment :) If they _are_ user-replaced values, fixing the format is still out of scope (it's existing content), so up to you when you get to it.

Reply:
yes they are literal, they do not change. They are the attribute keys. The attribute value contains the corresponding value.

Reply:
This settles it. Thank you both.